### ISSUE TITLE
[FEATURE] Improve accessibility

### ISSUE THREAD

**Issue by XhmikosR** [ColorlibHQ/AdminLTE#2743]
Currently many accessibility suggestions from upstream are not being followed in the demo pages.

This should be revisited later. For example, most if not all, of the icons should have `aria-hidden="true"` along with an `sr-only` block for screenreaders.

There are plenty of tools and this can be automated later. It's just that someone has to make the start and I could probably help automating the a11y checks if I have some time :)

**Comment by REJack** [comment]
That's a nice idea, I didn't know that there are tools for this. This is nice add for v3.1 😄 I'll start the v3.1 developing after your PR's.

**Comment by puikinsh** [comment]
Closing as part of issue triage for v4.0.0-rc7. This issue predates the current v4 architecture and is either resolved, outdated, or outside scope. Please open a new issue if the problem persists with the latest release.
